app/src/main/proto/vss.proto

```diff
@@ -0,0 +1,182 @@
+syntax = "proto3";
+option java_multiple_files = true;
+package org.vss;
+
+message GetObjectRequest {
+
+  /*
+    StoreId: All APIs operate within a single storeId.
+    StoreId is a keyspace identifier.
+    Ref: https://en.wikipedia.org/wiki/Keyspace_(distributed_data_store)
+    It is up to clients to use single or multiple stores for their use-case.
+    This can be used for client-isolation/ rate-limiting / throttling on the server-side.
+    Authorization and billing can also be performed at the storeId level.
+ */
+  string storeId = 1;
+
+  /*
+    Key for which the value is to be fetched.
+
+    Consistency Guarantee:
+    Get/Read operations against a key are consistent reads and will reflect all previous writes,
+    since Put/Write provides read-after-write and read-after-update consistency guarantees.
+
+    Read Isolation:
+    Get/Read operations against a key are ensured to have read-committed isolation.
+    Ref: https://en.wikipedia.org/wiki/Isolation_(database_systems)#Read_committed
+   */
+  string key = 2;
+}
+
+
+message GetObjectResponse {
+  /*
+    Fetched value and version along with the corresponding key in the request.
+   */
+  KeyValue value = 2;
+}
+
+
+message PutObjectRequest {
+
+  /*
+    StoreId: All APIs operate within a single storeId.
+    StoreId is a keyspace identifier.
+    Ref: https://en.wikipedia.org/wiki/Keyspace_(distributed_data_store)
+    It is up to clients to use single or multiple stores for their use-case.
+    This can be used for client-isolation/ rate-limiting / throttling on the server-side.
+    Authorization and billing can also be performed at the storeId level.
+ */
```

Comment:
Indentation of trailing "*/" seems inconsistent with the other fields. Could we use "//" instead? It's a bit more common.

Reply:
Done